source/code-snippets/usage-examples/deleteOne.ts

```diff
@@ -0,0 +1,26 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const movies = database.collection("movies");
+
+    const result = await movies.deleteOne({ title: { $type: "string" } });
```

Comment:
we don't think this TS example is necessary since it is identical to the `.js` one and there is no added value from TS over plain JS for this use case

Reply:
Thanks for the review! While I agree using TypeScript for this use case does not provide much value for readers, I think it still makes sense to include TS (or a note about TS) for this example. Here are my reasons for including it:

- Including TS communicates explicitly rather than implicitly that the TS is the same as the JS for this use case. I think if the TS example is removed readers will have to infer that a lack of TS means that there is no benefit to using TS here, which I think may be difficult for Jasmine persona (early developer) readers.
- Including TS provides a consistent experience across usage examples (all examples have a JS and TS component to them). I think the usage examples could look incomplete if some examples have a TS variant and some do not.

I agree that right now it is not clear that the TS and JS are identical and that readers receive no substantive benefits from using TS here. Do any of these options address this concern?

1. Include a comment in the TS example saying that the "TS and JS snippets are identical for this use case"
2. Add a note admonition above the code snippet saying "The TS and JS Snippets are Identical For this Use Case"
3. Include an admonition below examples where TS does not benefit readers saying, "TS does not provide significant value in this use case". 

Let me know if you disagree or would like to discuss this over slack!